src/contentScript/selectionTooltip/tooltipController.tsx

```diff
@@ -188,11 +236,19 @@ async function updatePosition(): Promise<void> {
       });
     },
   );
+
+  hideController.signal.addEventListener(
+    "abort",
+    () => {
+      cleanupAutoPosition();
+    },
+    { once: true },
+  );
```

Comment:
If you want to shorten this using `abort-utils`:

```suggestion
  onAbort(hideController, cleanupAutoPosition);
```